lib/internal/dom-exception.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const { internalBinding } = require('internal/bootstrap/loaders');
+const { registerDOMException } = internalBinding('messaging');
+
+const kName = Symbol('name');
+const kMessage = Symbol('message');
+
+const nameToCodeMap = new Map();
+
+class DOMException extends Error {
+  constructor(message = '', name = 'Error') {
+    super();
+    this[kMessage] = `${message}`;
+    this[kName] = `${name}`;
+    Error.captureStackTrace(this, DOMException);
```

Comment:
Is this required for stuff that `extends Error`?

Reply:
Let me double-check.